index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse</title>
+    <title>Microverse.</title>
 </head>
 <body>
     <h1>Hellow Microverse</h1>
```

Comment:
## 1. 
![pr-title (another copy)](https://user-images.githubusercontent.com/93039359/186141767-70970c31-914b-4edd-acbb-ba753a1d415b.png)


- [ ] Your PR has a title that is not so descriptive enough. Because **Adding of Script files** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Setup Hello Microverse** for your PR title. :+1: